lib/beacon_web/live/page_live.ex

```diff
@@ -26,7 +26,7 @@ defmodule BeaconWeb.PageLive do
 
     Beacon.PubSub.subscribe_page_update(site, path)
 
-    {:ok, socket}
+    if(Code.ensure_loaded?(Surface), do: {:ok, Surface.init(socket)}, else: {:ok, socket})
```

Comment:
Can you remove the surface stuff?
```suggestion
    {:ok, socket}
```